app/views/govuk_component/lead_paragraph.raw.html.erb

```diff
@@ -0,0 +1,13 @@
+<%
+  text ||= ""
+  def nbsp_between_last_two_words(text)
```

Comment:
This helper has some tests in government-frontend: https://github.com/alphagov/government-frontend/blob/master/test/helpers/typography_helper_test.rb – we should probably have something similar here in static.

Reply:
Added.